### DISCUSSION TITLE
Internal Links with nav section

### DISCUSSION THREAD

**Discussion by BaccanoMob** [mkdocs/mkdocs#3844] [Q&A]
Is it possible to use nav section to link between pages inside md file rather than manually setting file path? or are there any plugins to do this?

So if I'm moving pages in my docs folder, I do not have to worry about every single making changes in every mention of it other than ones in mkdocs.yml file.

**Comment by pawamoy** [comment]
Have a look at https://pypi.org/project/mkdocs-autorefs/.

**Reply by BaccanoMob** [reply]
Thanks for the reply! Ill look into it
